senza/cli.py

```diff
@@ -2047,6 +2047,12 @@ def scale_auto_scaling_group(asg, asg_name, desired_capacity):
             act.ok("NO CHANGES")
         else:
             kwargs = {}
+            if group["MinSize"] == 0 and desired_capacity > group["MinSize"]:
```

Comment:
This is a considerable change in the behavior. It also may result in some nasty side-effects for operations such as `senza scale 100`. Have you considered just logging the warning? That would already help operators to realize that `MinSize` is left at 0. A new flag, such as `--force` could then be used to do what you suggest. This would keep the backwards compatibility for any existing scripts but also give a hint to operators how to achieve the new desired behavior.

Reply:
Is there a way to update/patch the `MinSize` value of the ASG through senza? If yes, then we could add it to the message.

Reply:
What I was suggesting would be some sort of new flag for the scale command.
e.g. `senza scale --force foo 10`
The warning message would then suggest this flag whenever the MinSize was found at 0. That way the operator would know exactly what to do if that was intended. With time, the flag would become known along with its effects and new scripts that want the new behavior could just add the flag.

Reply:
Just realized that `--force` already exists for scale. It'd have to be another flag

Reply:
The scale command already contains a `--force` flag. It was up until now only used for updating more stacks than 1.

I would propose to use the flag also for this use case

Reply:
What do you think about `--minsize [value]`? It could work with or without arguments. With an argument it'd allow the operator to specify a minimum capacity different from the max size. Without the argument it'd use the same value as max size.

Reply:
I added the option `--min_size` that requires a value. I think this makes it the most explicit for users. 

What do you think?